tests/unit/sqlalchemy/test_compiler.py

```diff
@@ -54,6 +62,24 @@ def test_offset(dialect):
     assert str(query) == 'SELECT "table".id, "table".name \nFROM "table"\nOFFSET :param_1'
 
 
+def test_catalogs_argument(dialect):
+    statement = select(system_table)
+    query = statement.compile(dialect=dialect)
+    assert str(query) == 'SELECT "system".information_schema."table".id \nFROM "system".information_schema."table"'
+
+
+def test_catalogs_create_table(dialect):
+    statement = CreateTable(system_table)
+    query = statement.compile(dialect=dialect)
+    assert str(query) == \
+        '\n'\
+        'CREATE TABLE "system".information_schema."table" (\n'\
+        '\tid INTEGER NOT NULL, \n'\
+        '\tPRIMARY KEY (id)\n'\
```

Comment:
This DDL looks invalid as Trino query.

Reply:
I changed the table to use a fake `other` catalog so that this test doesn't potentially interfere with the `system` catalog.